.github/workflows/check-go-task.yml

```diff
@@ -25,8 +26,36 @@ on:
   repository_dispatch:
 
 jobs:
+  run-determination:
+    runs-on: ubuntu-latest
+    outputs:
+      result: ${{ steps.determination.outputs.result }}
+    steps:
+      - name: Determine if the rest of the workflow should run
+        id: determination
+        run: |
+          RELEASE_BRANCH_REGEX="refs/heads/[0-9]+.[0-9]+.x"
```

Comment:
Should we add the support for the `v` prefix here too?

Reply:
I think it's not necessary because we are looking for a release branch here, `v` prefixes will be added eventually to tags, so no need to modify this regex